Ruby/Ruby.sublime-syntax

```diff
@@ -117,10 +117,10 @@ contexts:
 
   class:
     # Defining a class method
-    - match: \b(class)\b(?=\s*<)
-      scope: keyword.control.class.ruby
-    - match: '\b(class)\b'
-      scope: meta.class.ruby keyword.control.class.ruby
+    - match: \bclass\b(?=\s*<)
+      scope: storage.type.class.ruby
+    - match: \bclass\b
+      scope: meta.class.ruby storage.type.class.ruby
```

Comment:
Via https://github.com/sublimehq/Packages/issues/582#issuecomment-453231444,
we decided on `keyword.declaration` for these types of keyword. `storage.type.class keyword.declaration` should be used in the transition period.

Reply:
Except D all syntaxes use `storage.type.class`. As stacking those keywords doesn't have any value, I prefer to stay with the majority of syntaxes which use `storage` only.

Reply:
So your argument is you don't want to use the new scope because the old one is still in use by other syntaxes?

Reply:
As noone haven't made any efforts to change it, yes.

Reply:
:man_shrugging: I'll note down to do a large PR changing various scope names to what we decided on in the past. In fact, I already took notes for preparing proper standards tracking for #1440. Hopefully I can manage this within the current dev cycle.

Reply:
@deathaxe Multiple contributors have gotten hung up on using `storage.type.…` for declaration keywords in the past. Part of the hang up is that many color schemes don't specify past `storage.type`, so `int` looks the same as `def`, and some languages will have both a keyword and a return type (if I recall correctly).

In an attempt to make a forward-looking change, but preserve not breaking all existing color schemes, the standard approach for now (which will be reflect in the naming docs when I do the next update) is to double up with `storage.type keyword.declaration`.